kubetools/kubernetes/config/__init__.py

```diff
@@ -182,6 +182,9 @@ def generate_kubernetes_configs_for_project(
             default_registry=default_registry,
         )
         app_annotations = copy_and_update(base_annotations)
+        for container_name, data in containers.items():
+            if data.get('annotations'):
+                app_annotations.update(data.get('annotations'))
```

Comment:
This is equivalent to (x3)
```suggestion
            app_annotations.update(data.get('annotations', {}))
```

Reply:
https://github.com/EDITD/kubetools/pull/165/commits/02c60076d29e2b4daaa303248cda84b8d7c7dd0b